src/components/elements/Modal/component.js

```diff
@@ -51,8 +63,12 @@ Component.propTypes = {
   children: PropTypes.node.isRequired,
   /** Does the modal have a close icon. */
   hasCloseIcon: PropTypes.bool,
+  /** Does the modal have padding around its content. */
+  hasPadding: PropTypes.bool,
   /** Does the modal have round corners. */
   hasRoundedCorners: PropTypes.bool,
+  /** The header of the modal when displayed. */
```

Comment:
This could be more explicit. Maybe...

```suggestion
  /** The header of the modal when displayed. The header is fixed at the top of the modal. */
```

or whatever you think is best.